apps/nextjs-website/src/lib/homepage.ts

```diff
@@ -86,16 +86,30 @@ export const makeHomepageProps = (
       ),
     },
   }),
-  productsShowcase: {
-    title: strapiHomepage.data.attributes.productsShowcase.title,
-    products: strapiHomepage.data.attributes.productsShowcase.products.data.map(
+  ecosystem: {
+    title: strapiHomepage.data.attributes.ecosystem.title,
+    productsTabName: strapiHomepage.data.attributes.ecosystem.productsTabName,
+    products: strapiHomepage.data.attributes.ecosystem.products.data.map(
       (product) => ({
-        name: product.attributes.name,
-        description: product.attributes.description,
-        slug: product.attributes.slug,
-        logo: product.attributes.logo.data.attributes,
+        title: product.attributes.name,
+        text: product.attributes.description ?? '',
+        href: product.attributes.slug,
+        icon: product.attributes.logo.data.attributes.url,
       })
     ),
+    solutionsTabName: strapiHomepage.data.attributes.ecosystem.solutionsTabName,
+    solutions: strapiHomepage.data.attributes.ecosystem.solutions.data.map(
+      (solution) => ({
+        title: solution.attributes.title,
+        text: solution.attributes.description ?? '',
+        href: solution.attributes.slug,
+        icon: solution.attributes.icon.data.attributes.url,
+      })
+    ),
+    solutionsCta: strapiHomepage.data.attributes.ecosystem.solutionsCta && {
+      variant: strapiHomepage.data.attributes.ecosystem.solutionsCta.variant,
+      link: strapiHomepage.data.attributes.ecosystem.solutionsCta.link,
+    },
   },
```

Comment:
As a consequence of the previous suggestion, you should manage the empty ecosystem here too:
```suggestion
  ...(strapiHomepage.data.attributes.ecosystem && {
    ecosystem: {
      title: strapiHomepage.data.attributes.ecosystem.title,
      productsTabName: strapiHomepage.data.attributes.ecosystem.productsTabName,
      products: strapiHomepage.data.attributes.ecosystem.products.data.map(
        (product) => ({
          title: product.attributes.name,
          text: product.attributes.description ?? '',
          href: product.attributes.slug,
          icon: product.attributes.logo.data.attributes.url,
        })
      ),
      solutionsTabName:
        strapiHomepage.data.attributes.ecosystem.solutionsTabName,
      solutions: strapiHomepage.data.attributes.ecosystem.solutions.data.map(
        (solution) => ({
          title: solution.attributes.title,
          text: solution.attributes.description ?? '',
          href: solution.attributes.slug,
          icon: solution.attributes.icon.data.attributes.url,
        })
      ),
      solutionsCta: strapiHomepage.data.attributes.ecosystem.solutionsCta && {
        variant: strapiHomepage.data.attributes.ecosystem.solutionsCta.variant,
        link: strapiHomepage.data.attributes.ecosystem.solutionsCta.link,
      },
    },
  }),
```